docker-compose.yaml

```diff
@@ -113,8 +113,8 @@ services:
       - INFERENCE_WORK_DIR=/mzai/lumigator/python/mzai/jobs/inference
       - RAY_DASHBOARD_PORT
       - RAY_HEAD_NODE_HOST
-      - MISTRAL_API_KEY
-      - OPENAI_API_KEY
+      - MISTRAL_API_KEY=${MISTRAL_API_KEY}
```

Comment:
@agpituk fyi

Reply:
@peteski22 related to your env injection issues from yesterday, this is helpful for these specific vars

Reply:
They should, but they don't the way we have it currently set - changing this specifically fixes this error at the env var level